images/build/Dockerfile

```diff
@@ -1,5 +1,9 @@
-FROM registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.23-openshift-4.19
+FROM registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.23-openshift-4.19 AS builder
 MAINTAINER CRC <devtools-cdk@redhat.com>
 
 WORKDIR $APP_ROOT/src
 COPY . .
+RUN make cross
+
+FROM registry.access.redhat.com/ubi9/ubi
+COPY --from=builder $APP_ROOT/src/out/ /opt/
```

Comment:
The binaries won’t be available under the same path in the image before/after this change. Is this going to be a problem for non-konflux users of this containerfile?